VisionDemo/clients/TFDemo/controllers/page/Start/widgets/RefreshButton/onClick.js

```diff
@@ -0,0 +1,50 @@
+	var canvas = $("#cameracanvas").get(0);    
+    var ctx = canvas.getContext("2d");
+    var img = new Image();
+    ctx.clearRect(0, 0, canvas.width, canvas.height);
+
+    var curr_doc = client.getWidget("CurrentDocText").text;
```

Comment:
Can you switch this to `currDoc`? You've used camelcase in other code for variable naming and that's more consistent with our variable naming than snake case.